pages/api/recipes.tsx

```diff
@@ -0,0 +1,14 @@
+import { connectToDatabase } from "../../lib/mongodb";
+
+export default async (req, res) => {
+  const { db } = await connectToDatabase();
+
+  const recipes = await db
```

Comment:
Maybe we should have the try, catch statement like on the article:

Example from the article:

async function getPosts(req,res){
    try {
        // connect to the database
        let { db } = await connectToDatabase();
        // fetch the posts
        let posts = await db
            .collection('posts')
            .find({})
            .sort({ published: -1 })
            .toArray();
        // return the posts
        return res.json({
            message: JSON.parse(JSON.stringify(posts)),
            success: true,
        });
    } catch (error) {
        // return the error
        return res.json({
            message: new Error(error).message,
            success: false,
        });
    }
}

Reply:
Nice one!
Done!